plugins/topology/README.md

```diff
@@ -81,6 +81,61 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
 
   ```
 
+- The following configuration must be added in`customResources` component in the [`app-config.yaml`](https://backstage.io/docs/features/kubernetes/configuration#configuring-kubernetes-clusters) file to view the edit code decorator:
+
+  ```yaml
+   kubernetes:
+     ...
+     customResources:
+        - group: 'org.eclipse.che'
+          apiVersion: 'v2'
+          plural: 'checlusters'
+  ```
+
+  Also, ensure that the `CheCluster` is granted a [`ClusterRole`](https://backstage.io/docs/features/kubernetes/configuration#role-based-access-control) as shown in the following example code:
+
+  ```yaml
+    ...
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: backstage-read-only
+    rules:
+      ...
+      - apiGroups:
+          - org.eclipse.che
+        resources:
+          - checlusters
+        verbs:
+          - get
+          - list
+
+  ```
+
+- The following annotations are added to workload resources, such as Deployments to navigate to the Git repository of the associated application using the edit code decorator:
+
+  ```yaml title="deployment.yaml"
+    ...
+    labels:
+      app.openshift.io/vcs-uri: <GIT_REPO_URL>
+  ```
+
+  You can also add the following annotation to navigate to a specific branch:
+
+  ```yaml title="deployment.yaml"
+      ...
+      labels:
+        app.openshift.io/vcs-ref: <GIT_REPO_BRANCH>
+  ```
+
+  > Note: If Red Hat OpenShift Dev Spaces is installed and configured and Git URL annotations are also present on the workload YAML file, then clicking on the edit code decorator redirects you to the Red Hat OpenShift Dev Spaces instance.
```

Comment:
Should we also add the link to the instructions for installing and configuring Openshift Dev Spaces ?